site/docs/themes/design-tokens/index.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Design tokens
+layout: DetailTechnical
+sidebar:
+  label: Overview
+  groupLabel: Design tokens
+  priority: 4
+---
+
+Salt styles its components and patterns using design tokens. Design tokens are name and value pairings that capture small, repeatable design choices—such as colors, fonts, and spacing—and are essential for storing key style attributes in a design system. They help maintain consistency across platforms and applications, making it easier to update and maintain a cohesive look throughout your products.
+
+<Image
+  src="/img/theming/name-value-pairing-light.png"
+  srcDark="/img/theming/name-value-pairing-dark.png"
+  alt=""
+/>
+
+## Why we use tokens
+
+There are several benefits to using design tokens instead of static values:
+
+- Consistency: Ensures visual harmony across products, boosting recognition and trust.
+- Efficiency: Accelerates design changes across platforms, saving time and effort.
+- Flexibility: Allows modifications without extensive code changes.
+- Scalability: Centralizes property definitions, streamlining the design system.
+
+Design tokens are adaptable building blocks that capture design decisions as name-value pairs. Their names and usage stay the same even if the values change, making it easier for designers and developers to work together and maintain consistency.
+
+<Image
+  src="/img/theming/token-themes-light.png"
+  srcDark="/img/theming/token-themes-dark.png"
+  alt=""
+/>
+
+## How tokens are structured
+
+Salt uses a tiered token structure to provide flexibility with mode, theme, and density options. The token tiers—Foundations, Palettes, and Semantic Characteristics—make it easy to adjust styles and settings across components, ensuring consistency and adaptability throughout your application.
+
+<Image
+  src="/img/theming/token-tiers-light.png"
+  srcDark="/img/theming/token-tiers-dark.png"
+  alt=""
+/>
+
+### Foundations
+
+Foundation tokens are the base-level tokens that reference the raw values at the core of the design system. For more information about each foundation, see the [Foundations](/salt/foundations/index) section.
+
+### Palettes
+
+Palette tokens are collections of commonly used foundation tokens and are referenced by semantic characteristic tokens. A palette token can point to different foundation tokens, allowing components to switch between light and dark modes.
+
+### Semantic characteristics
```

Comment:
We should say this as it is: Characteristic.
Then lead with Semantic tokens live in the Characteristic layer. They are groups of tokens that...